app/Actions/Database/StartDatabaseProxy.php

```
@@ -55,6 +56,18 @@ public function handle(StandaloneRedis|StandalonePostgresql|StandaloneMongodb|St
         if (isDev()) {
             $configuration_dir = '/var/lib/docker/volumes/coolify_dev_coolify_data/_data/databases/'.$database->uuid.'/proxy';
         }
+
+        // Get timeout from database config (supports nginx time format like '30s', '5m', '2h', '7d', or plain seconds)
+        $timeoutConfig = $database->public_proxy_timeout ?? '0';
+        
+        // Normalize the timeout value to nginx time format
+        $timeoutValue = ValidNginxTimeFormat::normalizeFormat($timeoutConfig);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Rules\ValidNginxTimeFormat' in method 'handle'. (undefined)

(StaticAccess)